src/Pyz/Yves/CatalogPage/CatalogPageConfig.php

```diff
@@ -11,6 +11,11 @@
 
 class CatalogPageConfig extends SprykerCatalogPageConfig
 {
+    /**
+     * @var bool
+     */
+    protected const IS_MINI_CART_ASYNC_MODE_ENABLED = true;
```

Comment:
Don't we need it in b2c-mp demoshop?

Reply:
`add-to-cart from catalog` is not integrated to MP demoshops.